internal/desktop_browser.py

```diff
@@ -359,6 +359,8 @@ def stop(self, job, task):
 
     def wait_for_idle(self, wait_time = 30):
         """Wait for no more than 50% CPU utilization for 400ms"""
+        if self.options.noidleall:
+            return
```

Comment:
This one makes me a bit nervous.  The system-wide idle checks shouldn't have any side-effects but skipping the idle wait after launching the browser might end up trying to automate something before the browser is actually ready.

It's worth trying since --noidle is mostly meant for local dev testing but this one is likely to have side-effects that might also impact dev testing.

Reply:
We could add a sleep with a const of x seconds, which would allow for consistency but also let the browser get ready.